staging/src/k8s.io/kubectl/pkg/validation/validation_test.go

```diff
@@ -32,14 +33,20 @@ import (
 
 var fakeSchema = testing.Fake{Path: filepath.Join("..", "..", "testdata", "openapi", "swagger.json")}
 
+type fakeResourcesGetter struct {
+	doc *openapi_v2.Document
+}
+
+func (r *fakeResourcesGetter) OpenAPISchema() (openapi.Resources, error) {
+	return openapi.NewOpenAPIData(r.doc)
+}
+
 var _ = Describe("resource validation using OpenAPI Schema", func() {
 	var validator Schema
 	BeforeEach(func() {
-		s, err := fakeSchema.OpenAPISchema()
-		Expect(err).ToNot(HaveOccurred())
-		resources, err := openapi.NewOpenAPIData(s)
+		doc, err := fakeSchema.OpenAPISchema()
 		Expect(err).ToNot(HaveOccurred())
-		validator = NewSchemaValidation(resources)
+		validator = NewSchemaValidation(&fakeResourcesGetter{doc: doc})
```

Comment:
That test does nothing though. Can we add a test that shows the fakeResourceGetter isn't called at all, maybe with a PanicingResourceGetter?

Reply:
we would have to create a validator for that, that might work.